src/openeo_gfmap/preprocessing/sar.py

```diff
@@ -1,9 +1,9 @@
 """Routines to pre-process sar signals."""
 
 import openeo
-from openeo.processes import array_create, if_, is_nodata, power
+from openeo.processes import array_create, power
 
-from openeo_gfmap import Backend, BackendContext
+from openeo_gfmap import BackendContext
 
 
 def compress_backscatter_uint16(
```

Comment:
I propose we make `backend_context` optional and set it to None by default.

We then add a warning for users who do set it:
```python
import warnings
if backend_context is not None:
    warnings.warn(
                "The `backend_context` argument is deprecated.", DeprecationWarning, stacklevel=2
            )
```

Reply:
that looks good. Can you push that?

Reply:
In a meeting now, so go ahead

Reply:
actually i don't know if that will work. Other repos don't use this as kwarg, and rely on the order of two args: `backend_context: BackendContext, cube: openeo.DataCube`.

We cannot simply make the first a kwarg without touching the second.

Reply:
I believe we should prioritize having a correct backscatter correction here, and not push any of the foreseen updates into this PR.